Makefile

```diff
@@ -15,7 +15,7 @@ build-image:
 
 .PHONY: run
 run:
-	cd ./cmd/proxy && go run . -config_file ../../config.dev.toml
+	go run ./cmd/proxy -config_file config-example.toml
```

Comment:
Let's keep this `config.dev.toml` so that we don't break people who already have links saved for reference as well as scripts that use the default one. It also says `.dev` so it should be clear that it's an "example" and not production ready. We can even add a comment at the top of the file to say `this is not for production` if that's your motivation behind the breaking change : )